src/components/Profile/ProfileInfoCard/index.js

```diff
@@ -141,266 +141,359 @@ const ProfileInfoCard = () => {
-                <p>
-                  <a href={profileData.website} target="_blank" rel="noopener noreferrer">
-                    <div
-                      style={{
-                        display: "flex",
-                      }}
+                    </a>
+                  </Grid>
+                )}
+                {checkAvailable(profileData.website) && (
+                  <Grid item xs={6}>
+                    <a
+                      href={profileData.website}
+                      target="_blank"
+                      rel="noopener noreferrer"
                     >
-                      <Box mr={1}>
-                        <LinkIcon fontSize="small" className="website-color" />
+                      <Box
+                        m={0.5}
+                        display="flex"
+                        flexDirection="row"
+                        alignItems="center"
+                      >
+                        <LinkIcon fontSize="large" className="website-color" />
+                        <p style={{ padding: "10px" }}>{profileData.website}</p>
                       </Box>{" "}
-                      {profileData.website}
-                    </div>
-                  </a>
-                </p>
-              )}
-              {checkAvailable(profileData.country) && (
-                <p className="mb-0">
-                  <a href={googleURI.concat(profileData.country)} target="_blank" rel="noopener noreferrer">
-                    <div
-                      style={{
-                        display: "flex",
-                      }}
+                    </a>
+                  </Grid>
+                )}
+                {checkAvailable(profileData.country) && (
+                  <Grid item xs={6}>
+                    <a
+                      href={googleURI.concat(profileData.country)}
+                      target="_blank"
+                      rel="noopener noreferrer"
                     >
-                      <Box mr={1}>
-                        <FlagIcon fontSize="small" className="website-color" />{" "}
-                      </Box>
-                      {profileData.country}
-                    </div>
-                  </a>
-                </p>
-              )}
+                      <Box
+                        m={0.5}
+                        display="flex"
+                        flexDirection="row"
+                        alignItems="center"
+                      >
+                        <FlagIcon fontSize="large" className="website-color" />
+                        <p style={{ padding: "10px" }}>{profileData.country}</p>
+                      </Box>{" "}
+                    </a>
+                  </Grid>
+                )}
+              </Box>
             </Box>
           </Grid>
-        </Grid>
+        </Box>
       </Card>
+
+      <Dialog
+        fullWidth
+        maxWidth="sm"
+        open={showImageDialog}
+        onClose={!showImageDialog}
```

Comment:
if i remember correctly, `onClose` prop accepts a function, here you provided a boolean value. Please double check.

Reply:
That's an error on my part: It should do something like : ```onClose={() => setShowImageDialog(false)}```

The same issue is present in [here](https://github.com/scorelab/Codelabz/blob/0919513330fd6491078a9d3c153f1917c5b00d27/src/components/Profile/ProfileInfoCard/index.js#L408) by the previous author.

Should I edit them both and update this Pull Request?